src/renderer/components/battle/Playerlist.vue

```diff
@@ -109,7 +109,6 @@ function dragEnterSpectators(event: DragEvent) {
 
 function dragStart(event: DragEvent, participant: Player | Bot) {
     if (isBot(participant)) {
-        if (isRaptor(participant) || isScavenger(participant)) return;
```

Comment:
Can you clarify what was broken here? This change breaks it for me:

1. Drag scav/raptor
2. Drop it to the team with player
3. Drop doesn't succeed, the UI stops being responsive, e.g. "add button", "join button" don't do anything

Another example
1. Drag scav/raptor
2. Drop to spectators
3. Spectators field gets the red background even after finishing drag operation